monitor_vitals.py

```diff
@@ -0,0 +1,43 @@
+from time import sleep
+import sys
+
+class Monitor_Vitals:
+  def is_temperature_ok(self, temperature):
+    if temperature in range(95, 102):
+      self.print_loading_message()
+      print('Temperature is normal')
+      return True
+    else:
+      self.print_loading_message()
+      print('Temperature critical!')
+      return False
+
+  def is_pulse_rate_ok(self, pulseRate):
+    if pulseRate in range(60, 100):
```

Comment:
The pattern of code is the same in all the `is_..._ok` functions. See if you can reduce the duplicated patterns.
Hint: Use a function like `is_parameter_ok(input_value, lower_limit, upper_limit)` Pass it a message also, if required